pkg/agent/cniserver/pod_configuration_windows.go

```diff
@@ -102,6 +103,44 @@ func (pc *podConfigurator) connectInterfaceToOVS(
 	return containerConfig, pc.connectInterfaceToOVSAsync(containerConfig, containerAccess)
 }
 
+func (pc *podConfigurator) configureInterfaces(
+	podName, podNamespace, containerID, containerNetNS string,
+	containerIFDev string, mtu int, sriovVFDeviceID string,
+	result *ipam.IPAMResult, createOVSPort bool, containerAccess *containerAccessArbitrator) error {
+	if !createOVSPort {
```

Comment:
On Windows, we should firstly create HNS Endpoint, then create OVS port. This is because when we firstly attach a container to the created HNS Endpoint ( provided in your current function `configureInterfacesCommon`), the vNIC is created, then we created OVS internal port to connect to the vNIC. If we firstly create OVS internal port, OVS would create a new vNIC, which is different from the one generated by HNSEndpoint, then container traffic can't go into OVS.